index.js

```diff
@@ -1,6 +1,9 @@
 require('dotenv').config();
 
 const express = require('express');
+const MongoClient = require('mongodb').MongoClient;
+const mongoose = require('mongoose');
+const student = require('./student.js');
```

Comment:
```suggestion
const mongoose = require('mongoose');
const student = require('./student.js');
```
Pour ce projet, tu ne dois pas utiliser mongoose